modules/SofaNonUniformFem/src/SofaNonUniformFem/HexahedronCompositeFEMForceFieldAndMass.inl

```diff
@@ -614,8 +614,8 @@ void HexahedronCompositeFEMForceFieldAndMass<T>::computeMechanicalMatricesDirect
                 for(int m=0; m<3; ++m)
                     for(int n=0; n<3; ++n)
                     {
-                        assembledStiffness.add( v1*3+m, v2*3+n, finestStiffnesses[i][j*3+m][k*3+n] );
-                        assembledMass.add( v1*3+m, v2*3+n, finestMasses[i][j*3+m][k*3+n] );
+                        assembledStiffness(v1 * 3 + m, v2 * 3 + n) += finestStiffnesses[i][j*3+m][k*3+n];
```

Comment:
I am surprised there is no change in the indices here. From some other of your changes, it seems indices in NewMat starts at 1, but here it seems to be 0...

Reply:
it is because it is using NewmatMat (from SOFA, so index from 0) and not NEWMAT::DenseMatrix (index from 1)

Reply:
ok thanks. I am relieved there is an explanation :)